cardinal/types/engine/info.go

```diff
@@ -0,0 +1,14 @@
+package engine
+
+type GetWorldResponse struct {
+	Namespace  string        `json:"namespace"`
+	Components []FieldDetail `json:"components"` // list of component names
+	Messages   []FieldDetail `json:"messages"`
+	Queries    []FieldDetail `json:"queries"`
+}
+
+type FieldDetail struct {
```

Comment:
1. Move this out of the `engine package. We don't want anything there anymore.
2. Comment what this is for

Reply:
done